triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/RunnerConfig.scala

```diff
@@ -26,6 +26,20 @@ object LogEncoder {
   object Json extends LogEncoder
 }
 
+sealed trait CompilerConfigBuilder extends Product with Serializable {
+  def build(majorLanguageVersion: LanguageMajorVersion): Compiler.Config
+}
+object CompilerConfigBuilder {
+  final case object Default extends CompilerConfigBuilder {
+    override def build(majorLanguageVersion: LanguageMajorVersion): Compiler.Config =
+      Compiler.Config.Default(majorLanguageVersion)
+  }
+  final case object Dev extends CompilerConfigBuilder {
+    override def build(majorLanguageVersion: LanguageMajorVersion): Compiler.Config =
+      Compiler.Config.Dev(majorLanguageVersion)
+  }
+}
+
```

Comment:
We can't simply have config field of type `LanguageMajorVersion => Compiler.Config` like we do in `Cli.scala` because the tests for this class check for equality of `RunnerConfig`s. So we use case objects to represent the two cases instead.